src/io/flutter/view/FlutterView.java

```diff
@@ -447,11 +459,18 @@ protected void waitForJxBrowserInstallation(FlutterApp app, InspectorService ins
   }
 
   protected void handleJxBrowserInstallationFailed(FlutterApp app, InspectorService inspectorService, ToolWindow toolWindow) {
-    // Allow user to manually restart.
-    presentClickableLabel(toolWindow, "JxBrowser installation failed. Click to retry.", (linkLabel, data) -> {
-      JxBrowserManager.getInstance().retryFromFailed(app.getProject());
-      handleJxBrowserInstallationInProgress(app, inspectorService, toolWindow);
-    });
+    if (System.getProperty(JxBrowserUtils.LICENSE_PROPERTY_NAME) == null) {
+      // If the license isn't available, allow the user to open the equivalent page in a non-embedded browser window.
+      presentClickableLabel(toolWindow, "The JxBrowser license could not be found. Click to open Devtools in the browser.", (linkLabel, data) -> {
+        presentDevTools(app, inspectorService, toolWindow, false);
+      });
+    } else {
+      // Allow the user to manually restart.
+      presentClickableLabel(toolWindow, "JxBrowser installation failed. Click to retry.", (linkLabel, data) -> {
```

Comment:
nit: try to avoid using the work "Click"/ "Click here" in messages. This could just be "Retry?"